projects/plugins/jetpack/extensions/shared/memberships/settings.js

```diff
@@ -367,3 +369,49 @@ export function NewsletterAccessPrePublishSettings( { accessLevel } ) {
 		/>
 	);
 }
+
+export function NewsletterEmailDocumentSettings() {
+	const isPostPublished = useSelect( select => select( editorStore ).isCurrentPostPublished(), [] );
+	const postType = useSelect( select => select( editorStore ).getCurrentPostType(), [] );
+	const [ postMeta, setPostMeta ] = useEntityProp( 'postType', postType, 'meta' );
+
+	const toggleSendEmail = value => {
+		const postMetaUpdate = {
+			...postMeta,
+			// Meta value is negated, "don't send", but toggle is truthy when enabled "send"
+			[ META_NAME_FOR_POST_DONT_EMAIL_TO_SUBS ]: ! value,
+		};
+		setPostMeta( postMetaUpdate );
+	};
+
+	const isSendEmailEnabled = useSelect( select => {
+		const meta = select( editorStore ).getEditedPostAttribute( 'meta' );
+		// Meta value is negated, "don't send", but toggle is truthy when enabled "send"
+		return ! meta?.[ META_NAME_FOR_POST_DONT_EMAIL_TO_SUBS ];
+	} );
+
+	/*
+	// Don't allow changing setting for a post that was already published in the past.
+	const postWasEverPublished = useSelect( select => {
+		const meta = select( editorStore ).getEditedPostAttribute( 'meta' );
+		return ! meta?.[ META_NAME_FOR_POST_WAS_EVER_PUBLISHED ];
+	} );
+	console.log( 'postWasEverPublished:', postWasEverPublished ); // eslint-disable-line
```

Comment:
This setting felt flaky, so commented it out. If it doesn't work, we can remove for now and add back in separate PR.